src/kernel/arch/x86/arch.zig

```diff
@@ -100,4 +125,73 @@ export nakedcc fn start_higher_half() noreturn {
     while (true) {}
 }
 
+///
+/// Initialise the architecture
+///
+/// Arguments:
+///
+/// Global changes:
+///
+/// Return:
+///
 export fn init() void {}
+
+///
+/// Inline assembly to write to a given port with a byte of data.
+///
+/// Arguments:
+///     IN port: u16 - The port to write to.
+///     IN data: u8  - The byte of data that will be sent.
+///
+/// Global changes:
+///
+/// Return:
+///
+export fn outb(port: u16, data: u8) void {
+    if (!is_test) {
```

Comment:
Why do we need to check if this is a test?

Reply:
This was here because when testing, you cant execute the OUT instruction, but now import a mock arch when testing, this can be removed. Good spot.